### PR TITLE
Update application copy

### PR DESCRIPTION
**Pull request by rachelcope**
Reason for Change
===================
* Update copy to make it more intuative/clear. 
* These changes were based on user feedback.
* https://trello.com/c/mxN3RiQK/289-expenses-update-copy-how-much-does-your-household-currently-pay-in-rent-or-mortgage-each-month
* https://trello.com/c/UD8WS1vf/296-expenses-update-copy-this-includes-arrearages-child-support-alimony-etc
* https://trello.com/c/P8kYZAT2/295-expenses-change-coolingto-air-conditioning
* https://trello.com/c/7UeCPd8V/291-signature-page-change-your-signature-to-sign-by-typing-your-full-legal-name

### CODE REVIEW

File: app/views/expenses_additional_sources/edit.html.erb [hunk 1]
```diff
@@ -40,7 +40,7 @@
         <%= f.mb_radio_set :court_ordered,
           "Does your household have court-ordered expenses?",
           [{ label: "Yes", value: true }, { label: "No", value: false }],
-          notes: "This includes child support, alimony, etc." %>
+          notes: "This includes arrearages, child support, alimony, etc." %>
```
Comment (jessieay) [inline app/views/expenses_additional_sources/edit.html.erb]: woah! what is an `arrearages` ?

**Comment by rachelcope** [comment]
[[SIGNATURE PAGE] Change "Your Signature" to "Sign by typing your full legal name"](https://trello.com/c/7UeCPd8V/291-signature-page-change-your-signature-to-sign-by-typing-your-full-legal-name)

**Comment by rachelcope** [comment]
[[EXPENSES] Change "Cooling"to "Air conditioning"](https://trello.com/c/P8kYZAT2/295-expenses-change-coolingto-air-conditioning)

**Comment by rachelcope** [comment]
[[EXPENSES] Update copy "This includes [arrearages], child support, alimony, etc.](https://trello.com/c/UD8WS1vf/296-expenses-update-copy-this-includes-arrearages-child-support-alimony-etc)

**Comment by rachelcope** [comment]
[[EXPENSES] Update copy "How much does your household [currently] pay in rent or mortgage each month?"](https://trello.com/c/mxN3RiQK/289-expenses-update-copy-how-much-does-your-household-currently-pay-in-rent-or-mortgage-each-month)

**Pull request merged**